sources/web/datalab/static/appbar.html

```diff
@@ -12,12 +12,12 @@
     <span class="autosave_status"></span>
   </span>
   <div class="btn-toolbar pull-right right-toolbar">
-    <div class="btn-group">
-      <button id="ungitButton" title="Open ungit on the notebooks repository" class="toolbar-btn">
+    <div id="ungitButton" class="btn-group">
```

Comment:
Synced with @jimmc on this. We might want to pick up some cleanup work later to make the code consistent.